tools/data-handler/test/command-handler-remove.test.ts

```diff
@@ -279,7 +282,11 @@ describe('remove command', () => {
       const calculateCmd = new Calculate();
       const removeCmd = new Remove(calculateCmd);
       await removeCmd
-        .remove(decisionRecordsPath, 'i-dont-exist', cardId)
+        .remove(
+          decisionRecordsPath,
```

Comment:
Should we actually remove this test? Now we're kind of testing what happens if a dev does not use the defined types.

Reply:
Yes. I can take it away. Let's have good tests only. :)